BraveWallet/Crypto/NetworkPicker.swift

```diff
@@ -91,22 +111,56 @@ struct NetworkPicker: View {
       )
       .clipShape(Capsule())
       .contentShape(Capsule())
-      .animation(nil, value: selectedNetwork)
+      .animation(nil, value: networkStore.selectedChain)
     }
     .accessibilityLabel(Strings.Wallet.selectedNetworkAccessibilityLabel)
-    .accessibilityValue(selectedNetwork.shortChainName)
+    .accessibilityValue(networkStore.selectedChain.shortChainName)
     .sheet(isPresented: $isPresentingAddNetwork) {
       NavigationView {
         CustomNetworkDetailsView(networkStore: networkStore, model: .init())
       }
     }
+    .alert(
+      isPresented: $isShowingNextNetworkAlert
+    ) {
+      Alert(
+        title: Text(String.localizedStringWithFormat(Strings.Wallet.createAccountAlertTitle, nextNetwork?.shortChainName ?? "")),
+        message: Text(Strings.Wallet.createAccountAlertMessage),
+        primaryButton: .default(Text(Strings.yes), action: {
+          // show create account for `nextNetwork.coin`
+          isPresentingAddAccount = true
+        }),
+        secondaryButton: .cancel(Text(Strings.no), action: {
+          // not creating account, don't switch to nextNetwork
+          self.nextNetwork = nil
+        })
+      )
+    }
+    .sheet(
+      isPresented: $isPresentingAddAccount
+    ) {
+      NavigationView {
+        AddAccountView(keyringStore: keyringStore) // TODO: pass `nextNetwork.coin` to account creation
+      }
+      .onDisappear {
+        Task { @MainActor in
+          guard let nextNetwork = nextNetwork else { return }
+          // iff it errors it's due to no accounts and we don't want to switch to nextNetwork
+          await networkStore.setSelectedChain(nextNetwork)
```

Comment:
iff :)

Reply:
Updated it while making other changes, but it's a [common short hand for if and only if](https://en.wikipedia.org/wiki/If_and_only_if) 🙂
[fa9183e](https://github.com/brave/brave-ios/pull/5681/commits/fa9183e24964c4e1c06e2382efbe259a852f7f8c)